docs/guides/addon-templates.md

```diff
@@ -0,0 +1,178 @@
+---
+page_title: "Addon Templates"
```

Comment:
As this is in global Guides section, seems use `CCE Addon Templates` is more clear.

Reply:
fixed